accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -62,9 +83,9 @@
     /**
      * Updates a specific patch version of an accessioned object. It does not create a new version / patch
      *
-     * @param accession
-     * @param version
-     * @param message
+     * @param accession Accession of the object that identify a particular model
+     * @param version Version number of the accessioned object
+     * @param message Type of the Accession model
```

Comment:
The `messages` does not represent the type of an accession object model. It represents the details of the object of type `MODEL`.

Reply:
`Details of the object of type MODEL` ?